config/config.go

```diff
@@ -396,6 +403,7 @@ type CertificatesInformationIndicators struct {
 // KialiFeatureFlags available from the CR
 type KialiFeatureFlags struct {
 	CertificatesInformationIndicators CertificatesInformationIndicators `yaml:"certificates_information_indicators,omitempty" json:"certificatesInformationIndicators"`
+	DisabledFeatures                  []string                          `yaml:"disabled_features,omitempty" json:"disabledFeatures,omitempty"`
```

Comment:
There should be control over what is populated in that list:
![image](https://user-images.githubusercontent.com/1662329/160806148-b6a545fc-02e3-4467-82d5-fc40b0ba457a.png)

Perhaps a fixed struct with flags would help instead of an open list.

But anyway, if the feature listed is not valid, message shouldn't be propagated and should be ignored.